reporter/src/funTest/kotlin/reporters/evaluatedmodel/EvaluatedModelReporterFunTest.kt

```diff
@@ -81,3 +78,5 @@ private fun TestConfiguration.generateReport(ortResult: OrtResult, options: Map<
 
     return EvaluatedModelReporter().generateReport(input, outputDir, options).single().readText()
 }
+
+private fun getAssetAsString(path: String): String = File("src/funTest/assets").resolve(path).readText()
```

Comment:
I guess I'd rather move the existing `getAssetAsString()` from `OsvServiceFunTest` to the `test-utils` module and use it from there; that should work as [all our library project have `test-utils` as a (fun)test dependency](https://github.com/oss-review-toolkit/ort/blob/6bff25b967a30d56c07c5328a6b2de3741dacccc/build.gradle.kts#L191-L193).

Reply:
Oh, I wasn't aware of the existence of this project. Thanks for pointing it out!